### DISCUSSION TITLE
[Combobox] Enter doesn't navigate to the item

### DISCUSSION THREAD

**Discussion by JJ810** [tailwindlabs/headlessui#1329] [Help]
Hey there!

I am using [Combobox](https://tailwindui.com/components/application-ui/forms/comboboxes#component-7674c1fe60b596923945f12306ccb59d) component for my search input and by pressing Enter from one of the combobox options, I want to navigate to some specific routes. And I couldn't solve this issue.

Please help me to fix it! Thanks in advance!

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

It's very hard to help you without looking at code. Please create a minimal reproduction repo or a CodeSandbox that shows the issue you are experiencing.

Once you've added that it will be easier for me or anyone else looking at this to help you!

**Comment by OliverRC** [comment]
To jump on this, I am curious how I override the default `Enter` key behavior.
I am using Vue and have tried a bunch of things, and I don't seem to be able to prevent the default behavior from firing.
